examples/bilby_nr_source_model.py

```
@@ -0,0 +1,669 @@
+    help="Extra f_min values to repeat the round-trip test at",
+)
+parser.add_argument(
+    "--f_max", type=float, default=1024.0, help="Maximum frequency (Hz)"
+)
+parser.add_argument("--no_plot", action="store_true", help="Skip the figures")
+args = parser.parse_args()
+
+sxs_id = args.sxs_id or SXS_PARTNER[args.gra_id]
+
+repo_path = (
+    subprocess.Popen(["git", "rev-parse", "--show-toplevel"], stdout=subprocess.PIPE)
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)